<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1269</width>
    <height>728</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QTabWidget" name="tabWidget">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="widget">
         <attribute name="title">
          <string>Tab 1</string>
         </attribute>
         <widget class="QWidget" name="verticalLayoutWidget">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>10</y>
            <width>1201</width>
            <height>361</height>
           </rect>
          </property>
          <layout class="QVBoxLayout" name="editTab">
           <item>
            <widget class="QTextEdit" name="codeEdit">
             <property name="styleSheet">
              <string notr="true">background-color: rgb(0, 0, 0);
color: rgb(255, 255, 255);
font: 9pt &quot;Arial&quot;;</string>
             </property>
             <property name="html">
              <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Arial'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="verticalLayoutWidget_2">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>400</y>
            <width>1201</width>
            <height>161</height>
           </rect>
          </property>
          <layout class="QVBoxLayout" name="outputTab">
           <item>
            <widget class="QTextBrowser" name="compileOutput">
             <property name="styleSheet">
              <string notr="true">background-color: rgb(0, 0, 0);
color: rgb(255, 255, 255);</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QPushButton" name="compileButton">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>570</y>
            <width>671</width>
            <height>21</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="text">
           <string>compile</string>
          </property>
         </widget>
         <widget class="QLabel" name="label">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>360</y>
            <width>1161</width>
            <height>51</height>
           </rect>
          </property>
          <property name="text">
           <string>output</string>
          </property>
         </widget>
         <zorder>label</zorder>
         <zorder>verticalLayoutWidget</zorder>
         <zorder>verticalLayoutWidget_2</zorder>
         <zorder>compileButton</zorder>
        </widget>
        <widget class="QWidget" name="tab_2">
         <attribute name="title">
          <string>Tab 2</string>
         </attribute>
         <widget class="QWidget" name="verticalLayoutWidget_3">
          <property name="geometry">
           <rect>
            <x>19</x>
            <y>19</y>
            <width>471</width>
            <height>531</height>
           </rect>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QGraphicsView" name="parseTree"/>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="verticalLayoutWidget_4">
          <property name="geometry">
           <rect>
            <x>499</x>
            <y>19</y>
            <width>711</width>
            <height>221</height>
           </rect>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QTextBrowser" name="Tab2Output">
             <property name="styleSheet">
              <string notr="true">background-color: rgb(0, 0, 0);
color: rgb(255, 255, 255);</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QLabel" name="label_3">
          <property name="geometry">
           <rect>
            <x>500</x>
            <y>-40</y>
            <width>211</width>
            <height>101</height>
           </rect>
          </property>
          <property name="text">
           <string> output</string>
          </property>
         </widget>
         <widget class="QWidget" name="verticalLayoutWidget_5">
          <property name="geometry">
           <rect>
            <x>499</x>
            <y>269</y>
            <width>711</width>
            <height>281</height>
           </rect>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="QTextBrowser" name="instruction">
             <property name="styleSheet">
              <string notr="true">background-color: rgb(0, 0, 0);
color: rgb(255, 255, 255);</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QLabel" name="label_4">
          <property name="geometry">
           <rect>
            <x>500</x>
            <y>190</y>
            <width>271</width>
            <height>121</height>
           </rect>
          </property>
          <property name="text">
           <string> instruction</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_2">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>-20</y>
            <width>161</width>
            <height>61</height>
           </rect>
          </property>
          <property name="text">
           <string>parse  tree</string>
          </property>
         </widget>
         <zorder>label_4</zorder>
         <zorder>label_3</zorder>
         <zorder>label_2</zorder>
         <zorder>verticalLayoutWidget_3</zorder>
         <zorder>verticalLayoutWidget_4</zorder>
         <zorder>verticalLayoutWidget_5</zorder>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1269</width>
     <height>17</height>
    </rect>
   </property>
   <widget class="QMenu" name="menufile">
    <property name="title">
     <string>file</string>
    </property>
    <addaction name="actionnew"/>
    <addaction name="actionsave"/>
    <addaction name="actionopen"/>
    <addaction name="actionexit"/>
   </widget>
   <widget class="QMenu" name="menubuild">
    <property name="title">
     <string>build</string>
    </property>
    <addaction name="actioncompile"/>
   </widget>
   <widget class="QMenu" name="menuhelp">
    <property name="title">
     <string>help</string>
    </property>
    <addaction name="actionabout"/>
    <addaction name="actiondoc"/>
   </widget>
   <addaction name="menufile"/>
   <addaction name="menubuild"/>
   <addaction name="menuhelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionnew">
   <property name="text">
    <string>new</string>
   </property>
  </action>
  <action name="actionsave">
   <property name="text">
    <string>save</string>
   </property>
  </action>
  <action name="actionopen">
   <property name="text">
    <string>open</string>
   </property>
  </action>
  <action name="actionabout">
   <property name="text">
    <string>about</string>
   </property>
  </action>
  <action name="actiondoc">
   <property name="text">
    <string>document</string>
   </property>
  </action>
  <action name="actioncompile">
   <property name="text">
    <string>compile</string>
   </property>
  </action>
  <action name="actionexit">
   <property name="text">
    <string>exit</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
